pkg/transferer/transferer.go

```diff
@@ -50,6 +50,19 @@ func (t Transferer) readChunksFromSource(ctx context.Context, p ChunkPool, chunk
 			log.Debug().Msg("Context is done, stopping chunks reading")
 			return ctx.Err()
 		default:
+			switch lc.GetLatestStatus() {
+			case LoadStatusWait:
+				time.Sleep(time.Second * 10) // TODO: make duration configurable
```

Comment:
there's no need to sleep here

delay is controlled by load checker so we can just recheck the latest status

and to avoid too many logs, change debug level to trace, when waiting